cap-ci/pipeline.template

```diff
@@ -1139,6 +1156,58 @@ jobs:
             {{- print $import_aks }}
             {{- end }}
             {{- print $stratos }}
+{{ end }}
+
+# The cluster will be cleared and deleted if `create-k8s` is true irrespective of the `clean-kubecf` value.
+{{- if or (index $config.jobs "clean-kubecf") (index $config "create-k8s") }}
```

Comment:
make clean already cleans cf. You don't need this job if create-k8s is true. But having it won't harm. Not sure if stratos and metrics cleanup was also added to `make clean`

Reply:
Agree having it won't harm unless it fails complaining in the cleanup stage. Also, it wouldn't matter if we are clearing up `stratos` and `metrics` if we deleting cluster anyways.